astropy/utils/system_info.py

```diff
@@ -0,0 +1,87 @@
+r"""
+A helper script to automate system info in bug reports.
+
+This can be run as
+```
+python -m astropy.utils.report_info
+```
+(note that the interpreter might be called python3 instead of python, depending
+on your system's details)
+
+This script should not require anything else than the standard library.
+"""
+
+__all__ = ["system_info"]
+
+import platform
+from importlib.metadata import version
+from importlib.util import find_spec
+
+
+def header(name: str) -> list[str]:
```

Comment:
I'd prefer if all the functions except `system_info()` would be explicitly marked as private:
```suggestion
def _header(name: str) -> list[str]:
```